packages/dev/docs/pages/blog/drag-and-drop.mdx

```diff
@@ -0,0 +1,244 @@
+date: 2022-11-10
+author: '[Devon Govett](https://twitter.com/devongovett)'
+---
+
+# Drag and drop
+
+We are excited to announce the release of drag and drop support in [React Aria](https://react-spectrum.adobe.com/react-aria/dnd.html) and [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/dnd.html)! This includes a suite of hooks for implementing drag and drop interactions, with support for both mouse and touch, as well as full parity for keyboard and screen reader input. We’ve designed these hooks with the following features in mind:
+
+- **Flexibility** – Our hooks include high level APIs for building common interactions such as dragging between lists, reordering, inserting, moving, copying, and file/directory uploading, as well as lower level APIs for building custom experiences.
+- **Accessibility** – Full support for keyboard and screen reader interactions is included out of the box, ensuring that applications implementing drag and drop using React Aria and React Spectrum are accessible with no additional work.
+- **Interoperability** – Drag data can be provided in multiple data formats for compatibility with many targets and external applications via the native HTML drag and drop API. Drag and drop integrates with multiple selection to allow dragging many objects at once.
+- **Customizability** – Interactions such as hit testing, keyboard navigation, and drop operations can be customized, as well as UIs for drag previews and drop indicators.
+
+<Video
+  src={heroVideo}
+  id="heroVideo"
+  muted
+  loop
+  autoPlay
+  tabIndex={0}
+  aria-label="Video showing features of the React Spectrum and React Aria drag and drop implementation"
+  style={{maxWidth: 'min(100%, 1920px)', display: 'block', margin: '40px auto'}} />
+
+## Introduction
+
+Drag and drop is a common UI interaction that allows users to transfer data between locations by directly moving a visual representation on screen. It is a flexible, efficient, and intuitive way for users to perform a variety of tasks, and is widely supported across both desktop and mobile operating systems.
+
+While drag and drop has historically been mostly limited to mouse and touchscreen users, keyboard and screen reader friendly alternatives are important for people who are not able to use these interaction methods. For example, copy and paste can often be used as an alternative to drag and drop. However, it is much more limited, and it can be hard to discover where pasting is accepted. This leaves applications to build custom UIs to accomplish the same tasks as drag and drop in an alternative way, which isn’t always easy to do, and often omitted leading to an inaccessible experience.
```

Comment:
I would say 'onus' is a less common word. I like the first iteration better. I do like the period after 'alternative way', it was getting a little long

Reply:
I'm fine w/ omitting the `onus` part, just felt that "This leaves applications to build..." sounded a bit awkward since it sounds like the application itself is performing the act of "building". Maybe "This puts the burden on" instead? Happy to keep the first iteration as well, I'm just nit picking here

Reply:
Yeah I agree with Rob. I made a slightly different version. lmk what you think of that one. 😄